cmd/markdown_help.go

```diff
@@ -0,0 +1,49 @@
+package cmd
+
+import _ "embed"
+
+//go:embed markdown/atmos_terraform_usage.md
+var terraform string
+
+//go:embed markdown/atmos_terraform_plan_usage.md
+var terraformPlan string
+
+//go:embed markdown/atmos_terraform_apply_usage.md
+var terraformApply string
+
+//go:embed markdown/atmos_workflow_usage.md
+var workflow string
+
+//go:embed markdown/atmos_about_usage.md
+var atmosAbout string
+
+type ExampleContent struct {
+	Content    string
+	Suggestion string
+}
+
+var doubleDashHint string = "Use double dashes to separate Atmos-specific options from native arguments and flags for the command."
+
+var examples map[string]ExampleContent = map[string]ExampleContent{
```

Comment:
Here we all them examples, but the files are called usage. Let's make it consistent one way or another